src/main/java/com/robertx22/age_of_exile/database/data/StatMod.java

```diff
@@ -170,6 +171,11 @@ public List<Component> getEstimationTooltip(int lvl) {
 
         list.add(txt);
 
+        var info = new TooltipInfo();
+        if(info.shouldShowDescriptions()) {
+            list.addAll(GetStat().getCutDescTooltip());
```

Comment:
add stat desc show for getEstimationTooltip(), It means some items like Rune can now press Alt to show the stat desc.